src/V3EmitCSyms.cpp

```diff
@@ -864,6 +864,13 @@ std::vector<std::string> EmitCSyms::getSymDtorStmts() {
         add("_vm_pgoProfiler.write(\"" + topClassName()
             + "\", _vm_contextp__->profVltFilename());");
     }
+    add("// Tear down scopes");
+    for (const auto& itpair : m_scopeNames) {
+        const ScopeData& sd = itpair.second;
+        const std::string id = protect("__Vscopep_" + sd.m_symName);
+        add("VL_DO_DANGLING(delete " + id + ", " + id + ");");
```

Comment:
```suggestion
        add("VL_DO_CLEAN(delete " + id + ", " + id + ");");
```
Since we want the user to observe that the pointer is now null, versus only doing so for debug safety.  Granted they shouldn't be looking at it, as we're in the cleanup phase, but anyhow.